docs/integrations/destinations/kafka.md

```diff
@@ -0,0 +1,87 @@
+# Kafka
+
+## Overview
+
+The Airbyte Kafka destination allows you to sync data to Kafka. Each stream is written to the corresponding Kafka topic.
+
+### Sync overview
+
+#### Output schema
+
+Each stream will be output into a Kafka topic.
+
+Currently, this connector only writes data with JSON format. More formats (e.g. Apache Avro) will be supported in the future.
+
+Each record will contain in its key the uuid assigned by Airbyte, and in the value these 3 fields:
+
+* `_airbyte_ab_id`: a uuid assigned by Airbyte to each event that is processed.
+* `_airbyte_emitted_at`:  a timestamp representing when the event was pulled from the data source.
+* `_airbyte_data`: a json blob representing with the event data.
+
+#### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | No |  |
+| Incremental - Append Sync | Yes |  |
+| Namespaces | Yes |  |
+
+## Getting started
+
+### Requirements
+
+To use the Kafka destination, you'll need:
+
+* A Kafka cluster 1.0 or above.
+
+### Setup guide
+
+#### Network Access
+
+Make sure your Kafka brokers can be accessed by Airbyte.
+
+#### **Permissions**
+
+Airbyte should be allowed to write messages into topics, and these topics should be created before writing into Kafka
```

Comment:
what does the user need to do in order to allow access for the connector?

Reply:
The user shouldn't do anything but just configure how they want to deliver the messages based on the message delivery guarantees. Topics should be configured by the Kafka admins